src/coreclr/vm/stackwalk.cpp

```diff
@@ -3161,18 +3140,6 @@ void StackFrameIterator::PostProcessingForManagedFrames(void)
         m_frameState = SFITER_NATIVE_MARKER_FRAME;
         m_crawl.isNativeMarker = true;
     }
-#ifdef TARGET_X86
-    else if (hasReversePInvoke)
```

Comment:
I have added this code as a fix to a bug, the fix was https://github.com/dotnet/runtime/pull/58796. I think that switching to handling the skipped the `InlinedCallFrame` on x86 like on any other platform should make my old fix unnecessary, however I'd suggest verifying it with the test that was reproducing the issue.
The repro was:
```
set COMPlus_TieredCompilation=0
set COMPlus_JitStress=71
Interop\UnmanagedCallersOnly\UnmanagedCallersOnlyTest\UnmanagedCallersOnlyTest.cmd
```
It is possible though that the repro is obsolete.

Reply:
I was also trying to find some historical info on why the x86 had this different order of reporting the `InlinedCallFrame` and its containing managed frame, but haven't found one.

Reply:
It actually looks like the settings to get a reliable repro of the issue were more involved:
```
set COMPlus_JitStressModeNamesOnly=1
set COMPlus_JitStressModeNames=STRESS_BB_PROFILE STRESS_LEGACY_INLINE
set COMPlus_JitStressRange=38891b70 566ce885
set COMPlus_TieredCompilation=0
```